packages/storybook/context/http.context.js

```diff
@@ -1,9 +1,9 @@
 import Vue from 'vue';
-import http from '@justeat/f-http';
+import httpModule from '@justeat/f-http';
 
 // Refer to f-http documentation for usage
 export default () => {
-    const httpClient = http.createClient();
+    const httpClient = new httpModule.CreateClient();
```

Comment:
Could we not export the http as a const also so we don't have to pass it through to service functions?

Reply:
How do you mean? Making it a global?

Reply:
thanks for the explanation, as per our convo makes mpre sense.